edited "deer family"

cmrn-rhi opened the pull request:
- added Exact Synonym: cervid
- added definition
- added image
- added taxon relationship with 'Cervidae'

cmrn-rhi committed:
edit "deer family"

- added Exact Synonym: cervid
- added definition
- added image
- added taxon relationship with 'Cervidae'

ddooley commented:
has problem of nodeid.  Could try resaving "foodon-edit.owl" as "rdf/xml syntax"

cmrn-rhi committed:
Resaved foodon-edit.owl rdf/xml syntax, minus "cervid" from initial push

Resaved foodon-edit.owl file over top of the existing one via “save as …” “rdf/xml syntax”.
Passed ""nodeid"" check (null).

- removed exact synonym ""cervid"" since it was added in #95 (branch: source-rhiannon-13)
